mass_mailing_custom_unsubscribe/readme/CONFIGURE.rst

```diff
@@ -5,3 +5,12 @@ they are going to unsubscribe. To do it:
 #. Create / edit / remove / sort as usual.
 #. If *Details required* is enabled, they will have to fill a text area to
    continue.
+
+For having discrete unsubscriptions from other recipients than the mailing
+lists, you need to add a glue module that adds 2 fields in the associated
+model:
```

Comment:
"discrete" is not too easy to understand, and maybe the model already has those fields. I'd reword this paragraph like this:

```suggestion
To allow managing unsubscriptions per record for other recipients than the
mailing list contacts, the associated model needs to include these fields:
```

Reply:
I know that for us Spanish people, it sounds weird, as we tend to translate it as `discreto`, which relates to someone cautious, polite, etc, but seeing the meaning, it seems the exact term:

https://dictionary.cambridge.org/es/diccionario/ingles/discrete
https://dictionary.cambridge.org/es/diccionario/ingles/discretely

Reply:
It always reminds me of this Futurama joke
![image](https://user-images.githubusercontent.com/5040182/85298044-dc7bef00-b4a3-11ea-8eab-6bb78bcdfbdd.png)